pkg/errors/errors_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package errors
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/cli-utils/pkg/apply/taskrunner"
+	"sigs.k8s.io/cli-utils/pkg/inventory"
+	"sigs.k8s.io/cli-utils/pkg/kstatus/status"
+	"sigs.k8s.io/cli-utils/pkg/object"
+)
+
+func TestTextForError(t *testing.T) {
+	testCases := map[string]struct {
+		err             error
+		cmdNameBase     string
+		expectFound     bool
+		expectedErrText string
+	}{
+		"kapply command base name": {
+			err:             inventory.NoInventoryObjError{},
+			cmdNameBase:     "kapply",
+			expectFound:     true,
+			expectedErrText: "Please run \"kapply init\" command.",
+		},
+		"kpt live command base name": {
```

Comment:
This is sigs-cli project and kpt should not be referred here.

Reply:
We should add tests in kpt for this scenario.

Reply:
Removed reference to kpt